### PR TITLE
Example configuration for common validators

### PR DESCRIPTION
**Pull request by RobinKnipe**
Added a more detailed description of the `exactlength` validator configuration to provide a template for using the common validators that require extra configuration. Configuration examples for both the fields and error token setup are given. There is also a slightly more in depth look at how the `before` validator could be used.

### CODE REVIEW

**Comment by RobinKnipe** [comment]
@gavboulton : I couldn't find an example of how to configure the validators that require more than just their name, so I've made some changes to the README. Please review.

File: README.md [hunk 1]
```diff
@@ -56,7 +56,46 @@ These methods are synchronous and take only the request and response obejct as a
...
+            type: 'exactlength',
+            arguments: [ 9 ]
+        }
+    ]
+}
+```
+
+Note: The object notation for the `exactlength` validator. This allows the definition of the extra coniguration property `arguments`. In the example the exact length is set to `9`. Some of the more complex validators - like `before` and `after` - may require several parameters, e.g. validating a date was before 1 year, 2 months, 3 days, 4 hours, 5 minutes ago, would look like: `arguments: [1, 'years', 2, 'months', 3, 'days', 4, 'hours', 5, 'minutes']`
+
+##### Exact length error token configuration
```
Comment (daniel-ac-martin) [inline README.md]: This section should probably live either with the wizard documentation or the HOF documentation. The concept of individual error messages doesn't exist at this level.

**Comment by daniel-ac-martin** [comment]
Besides my line comment this LGTM.

**Comment by daniel-ac-martin** [comment]
@easternbloc: Possibly we should have something similar in the HOF docs or an example in the example form.

File: README.md [hunk 2]
```diff
@@ -56,7 +56,46 @@ These methods are synchronous and take only the request and response obejct as a
...
+        'numeric',
+        {
+            type: 'exactlength',
+            arguments: [ 9 ]
+        }
+    ]
+}
+```
+
+Note: The object notation for the `exactlength` validator. This allows the definition of the extra coniguration property `arguments`. In the example the exact length is set to `9`. Some of the more complex validators - like `before` and `after` - may require several parameters, e.g. validating a date was before 1 year, 2 months, 3 days, 4 hours, 5 minutes ago, would look like: `arguments: [1, 'years', 2, 'months', 3, 'days', 4, 'hours', 5, 'minutes']`
```
Comment (HughePaul) [inline README.md]: Misspelling of "configuration"

**Comment by gavboulton** [comment]
Closing due to inactivity.

**Pull request closed without merging**